elrond-wasm-debug/src/meta/meta_generate_snippets.rs

```diff
@@ -0,0 +1,439 @@
+use std::{
```

Comment:
I think it's already enough code that turning this module into a folder with several source files is in order.